tools/codegen/gen.py

```diff
@@ -897,7 +897,7 @@ def compute_registration_declarations(f: NativeFunction) -> str:
     comment_data : Dict[str, str] = {
         'schema': f'aten::{f.func}',
         'dispatch': str(f.dispatch is not None),
-        'math': str(f.dispatch is not None and 'Math' in f.dispatch)
+        'default': str(f.dispatch is not None and ('Math' in f.dispatch or 'DefaultBackend' in f.dispatch))
```

Comment:
Is there a paired XLA change? Or was math not being used yet.

Reply:
Yea there's a paired XLA change

Reply:
This is re-triggering my sense that `['Math', 'DefaultBackend']` is a semantically significant grouping and that we should define it somewhere (maybe not model.py, but somewhere), per the review comment in #45719. I don't know if it's urgent to do before landing this PR, but I'd like us to understand what common property it is that causes these two keys to show show up together, and ideally capture it in a definition.